pkg/api/v20240812preview/openshiftcluster_validatestatic.go

```diff
@@ -499,5 +499,10 @@ func (sv openShiftClusterStaticValidator) validatePlatformIdentities(oc *OpenShi
 	if clusterIdentityPresent != operatorRolePresent {
 		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "identity", "Cluster identity and platform workload identities require each other.")
 	}
+
+	if operatorRolePresent && len(oc.Identity.UserAssignedIdentities) != 1 {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "identity", "The provided set of user assigned identities is invalid; there should be exactly one.")
```

Comment:
Is there any documentation link we should be returning to users here?

Reply:
I used the following usage of `support ticket`  when returning for other error in the backend:- https://github.com/Azure/ARO-RP/blob/master/pkg/util/steps/condition.go#L120
I can reach region leads for the better framing/detailing of this error message.

Reply:
Done!! Thanks.